src/main/resources/schema.sql

```diff
@@ -15,3 +15,51 @@ CREATE TABLE IF NOT EXISTS aircraft (
     routing_range numeric NOT NULL,
     name varchar(25)
 );
+
+CREATE TABLE IF NOT EXISTS airport(
+    airport_id char(3) PRIMARY KEY,
+    a_name varchar(25) NOT NULL,
+    a_city varchar(25) NOT NULL,
+    a_country varchar(25) NOT NULL,
+    flight_number varchar(6) NOT NULL
+);
+
+CREATE TYPE wifi_enum AS ENUM ('paid' , 'free', 'no');
+
+CREATE TABLE IF NOT EXISTS services(
+	service_id serial PRIMARY KEY,
+    movie varchar(25),
+    wifi wifi_enum,
+    meal varchar(25)
+);
+
+CREATE TABLE IF NOT EXISTS employee(
+	employee_id serial PRIMARY KEY,
+    aircraft_id serial REFERENCES aircraft,
+    aiport_id serial REFERENCES airport,
+    position_id serial REFERENCES position,
```

Comment:
@gheqimi changed the name of position to job since position is a key word so position needs to be changed accordingly The table looks like this now.

```
CREATE TABLE IF NOT EXISTS job(
    job_id serial PRIMARY KEY,
    job_type varchar(25) NOT NULL,
    salary numeric(6) NOT NULL
);
```